next/components/ui/Breadcrumbs/MobileBreadcrumbs.tsx

```diff
@@ -54,10 +58,10 @@ const MobileBreadcrumbs = ({ breadcrumbs }: BreadcrumbsProps) => {
               // eslint-disable-next-line react/no-array-index-key
               <li className="text-small font-medium" key={index}>
                 {breadcrumb.path ? (
-                  <MLink href={breadcrumb.path} variant="underlined" className="flex gap-1">
+                  <Button href={breadcrumb.path} className="flex gap-1">
```

Comment:
MLink is kay here.
Moreover, you didn't provide underline style for the Button.